extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js

```diff
@@ -119,7 +119,7 @@ const FADE_CURVE = bezierCurve(0.8, 0, 0.2, 1);
 const MAX_TRANSITION_DURATION = 1000; // ms
 const MIN_TRANSITION_DURATION = 500; // ms
 const MAX_DISTANCE_APPROXIMATION = 250; // px
-const MOTION_DURATION_RATIO = 0.8; // fraction of animation
+const MOTION_DURATION_RATIO = 0.5; // fraction of animation
```

Comment:
This affects all close animations and was a number @spacedino came up with. I don't think we want to change it for all closes animations. Perhaps just for swipe to dismiss?

Reply:
I softened the closing animation curve, which spreads the motion more evenly out over the duration, and therefore feels slow(er) with the existing constants. My solution was to make this number smaller to speed the animation up.

What other close animations are affected by the constants here?

Reply:
MOTION_DURATION_RATIO affects the open animation as well as the normal close animation (hitting the 'x' button).

Reply:
I see it now. The open/close animations are definitely nicer with the existing value.

Looking at it again, I see what I was originally going for by changing this value (it's *snappy!*), but I also don't mind the swipe-to-dismiss at the existing value. Less snappy, but maybe even slightly more luxurious, like how a heavy, well-insulated car door takes longer to close. SO I'm fine reverting this line or breaking out this constant into two numbers.

Also cc @nainar so she can heckle me if I don't respond to this for a few days again. :)